test/integration/answer_test.rb

```diff
@@ -1,6 +1,8 @@
 require 'test_helper'
 
 class AnswerTest < ActionDispatch::IntegrationTest
+  include RummagerHelpers
```

Comment:
Is this needed here? Doesn't look like we're using it?

Reply:
I'll check that, thanks

Reply:
Removed, thanks